src/orwell/callbacks/ProcessHello.cpp

```diff
@@ -86,12 +86,20 @@ void ProcessHello::execute()
 				m_publisher->send(aReply);
 				aFailure = false;
 
+				if (aHelloMessage.has_ready())
+				{
+					ORWELL_LOG_INFO("Hello ready ? " << aHelloMessage.ready());
```

Comment:
Whitespace typo: "Hello ready?"

Reply:
Well, it is not a typo. I really prefer to have a space there.